site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -8,6 +8,30 @@ NOTES FOR CONTENT CREATORS
 - Replace post tile, date, description and link to you post.
 ////
 
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Cassandra Day + Wakanda Forever
+[discrete]
+==== October 22, 2022
+------
+[openblock,card-content]
+------
+Join us for an action-packed Cassandra Day in Santa Clara, Bellevue & Houston with talks, workshop and a private screening of Wakanda Forever!
+
+[openblock,card-btn card-btn--blog]
+--------
+[.btn.btn--alt]
+xref:blog/Cassandra-Day-SC-Bellevue-Houston-WakandaForeverCassandra-Day-SC-Bellevue-Houston-WakandaForeveradoc[Read More]
```

Comment:
this is broke @ErickRamirezAU 

missing dot before adoc, and the name is completely whack.

fixed w/ https://github.com/apache/cassandra-website/commit/5e64c4822a489b5532892a38219a666234ba16b3